src/test/java/com/notably/logic/parser/NotablyParserTest.java

```diff
@@ -0,0 +1,53 @@
+package com.notably.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.logic.commands.Command;
+import com.notably.logic.commands.DeleteCommand;
+import com.notably.logic.commands.NewCommand;
+import com.notably.logic.commands.OpenCommand;
+
+
+public class NotablyParserTest {
```

Comment:
Let's add more test cases here? In particular, let's test the invalid cases thoroughly since parser is essentially the first point of contact with the user! A user can input any kind of faulty commands and we should catch it!

Reply:
Oh yes, will do further testing and catch all unexpected errors in the later commits.

Reply:
Alright!